internal/task/init_rainbond_cluster.go

```diff
@@ -81,6 +82,21 @@ func (c *InitRainbondCluster) Run(ctx context.Context) {
 			return
 		}
 	}
+	splitVersions := strings.Split(cluster.KubernetesVersion, ".")
```

Comment:
check version by function of this package: `k8s.io/apimachinery/pkg/util/version`